### ISSUE TITLE
Installed Recent Houdini For V-Ray Beta. Can't Find V-Ray Shelf.

### ISSUE THREAD

**Issue by MichaelWaisJr** [ChaosGroup/vray-for-houdini#318]
Hi all,

I installed the most recent Houdini for V-Ray beta. I ran hfs16.5.536.bat, which opens that version of Houdini just fine, but I can't for the life of me find the V-Ray shelf when I hit the "+" sign in Houdini.

I'm using Houdini Indie 16.5.536, if it matters, and I've downloaded the V-Ray for Houdini 16.5.536 version.

Can anybody please tell me what steps I may have missed in the installation?

Thanks in advance for any help.

-Michael Wais

**Comment by bdancer** [comment]
Hello, the main reason is that probably you have modified variables in your houdini.env (you may have it edited if you've installed any additional plugins to Houdini).

Make sure your variables in houdini.env are including environment variables.

I've posted some details here https://forums.chaosgroup.com/forum/v-ray-next-for-houdini-beta/v-ray-next-for-houdini-beta-problems/1006766-vray-beta-not-working-unless-i-delete-my-regular-houdini-env.

Basically in your houdini.env you need to have smth like this:
```
HOUDINI_PATH = "c:\some\custom\path;${HOUDINI_PATH};&"
```

**Comment by bdancer** [comment]
If this won't help drop me an email at andrei.izrantcev@chaosgroup.com to organize some remote access.

**Comment by MichaelWaisJr** [comment]
Thanks B! I'll give it a shot and e-mail you if that doesn't work. :)
